general/getting-started-in-tech.md

```diff
@@ -0,0 +1,99 @@
+- You can see results very fast.
+- They are ubiquitous: a lot of technologies that started on the web are now used in many other places. Even if you end up working in a domain that has nothing to do with the web, you will likely use or interact with web technologies.
+
+Web technologies are designed in such a way that they tend to more clearly separate user interface work from data processing work. As a result, web developer jobs typically fall into three broad categories:
+- Front end: work that deals with user interfaces
+- Back end: work that deals with data processing
+- Full stack: work that combines both aspects
+
+Both front-end and back-end have their own complexities and neither is easier than the other. So let’s have a look at what they both involve. Take this web page as an example. It is a page on the codebar.io website and you are reading it via a web browser on a computer or mobile device. The codebar website runs on a computer in a data center that runs a piece of software called a web server. Both talk to each other via the internet.
+
+To build a website like this involves two pieces of code:
+- The front-end code that runs in the web browser
+- The back-end code that runs on the web server
+
+Below we got into more depth about these roles, and what languages you would need to learn.
+
+## Front-end Development
+
+As a front-end developer you would be responsible for the code that runs in the web browser and displays information to visitors. It is typically the practice of writing HTML, CSS and JavaScript.
+
+#### HTML (Hyper Text Markup Language)
+
+HTML is the language used to build websites. All text and content that you see on the internet is built using HTML.
+
+#### CSS (Cascading Style Sheets)
+
+CSS is used alongside HTML to style the page and make it look good.
+
+
+#### JavaScript
+
+JavaScript is used to change what is displayed on a web page in response to user activity.
+
+## Back-end Development
+
+As a back-end developer you would be responsible for the code that runs on the server. This type of development usually consists of three parts: a server, an application, and a database. Many back-end developers know front-end languages such as HTML and CSS but need to use languages such as Ruby on Rails, Python, PHP, Java, and .NET to get the job done.
+
+Front-end and Back-end development overlap a fair amount so starting in one role doesn’t mean that you can't switch or learn the other, should you change your mind further down the line. They also have strong ties to other areas of technology. For instance, front-end development shares a lot with mobile app development; back-end development with data science.
+
+#### Ruby
+Ruby is an object oriented scripting programming language typically used for building web applications. The most common implementation of Ruby is Rails web, the development framework built with Ruby.
```

Comment:
@KimberleyCook I think we should use the full name of the framework here, e.g. Ruby on Rails. How about rewording this sentence like this?

```suggestion
Ruby is an object oriented scripting programming language typically used for building web applications. It is most commonly used with Ruby on Rails, a web development framework built with Ruby.
```